rpmbuild/copr_rpmbuild/helpers.py

```diff
@@ -366,3 +378,47 @@ def is_srpm_build(task):
     Return `True` if the `self.source_dict` belongs to a SRPM build task
     """
     return task.get("source_type") is not None
+
+
+class Spec:
+    """
+    Wrapper around `specfile.Specfile` to easily access attributes that we need
+    """
+
+    def __init__(self, path):
+        try:
+            macros = [("fedora", "39"), ("rhel", "")]
```

Comment:
We should loop over all chroots in the future. For now, we should have `macros = []` and put a TODO into the code.

Reply:
Updated